tests/test__pycrypto_crypt.py

```diff
@@ -23,7 +23,7 @@
 class TestPyCryptoVerifier(unittest.TestCase):
 
     PUBLIC_KEY_FILENAME = os.path.join(os.path.dirname(__file__),
```

Comment:
Does the name of this constant need to change the way the name of the file is changing?